staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -965,6 +965,8 @@ type PodSecurityPolicySpec struct {
 	// e.g. "foo.*" forbids "foo.bar", "foo.baz", etc.
 	// +optional
 	ForbiddenSysctls []string `json:"forbiddenSysctls,omitempty" protobuf:"bytes,20,rep,name=forbiddenSysctls"`
+	// runAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
+	RunAsGroup RunAsGroupStrategyOptions `json:"runAsGroup" protobuf:"bytes,21,opt,name=runAsGroup"`
```

Comment:
because this is a new field, this has to be optional, which means it must also be a pointer, and marked omitempty (comment applies to other types files as well)

Reply:
we also need to determine what the default behavior is when this is nil

Reply:
The default behavior must be "RunAsAny", otherwise existing policies will start rejecting pods that were previously allowed.